Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/server/DccValidationServer.kt

```diff
@@ -81,6 +97,29 @@ class DccValidationServer @Inject constructor(
         }
     }
 
+    private fun ResponseBody.parseBody() = parseBody(byteStream())
+
+    @VisibleForTesting
+    internal fun parseBody(inputStream: InputStream): String {
```

Comment:
Can parseBody also be used for the countries?

Reply:
there sure is a lot of duplicate code!